### PR TITLE
Wiki Reorg: Nominator Page

### PR DESCRIPTION
**Pull request by filippoweb3**
See [this issue](https://github.com/w3f/polkadot-wiki/issues/4376) and [this google doc](https://docs.google.com/document/d/1Fl8ReBIzcaQEOyF6s03d7MEcgk8aTVEt7xjmzAk_k5I/edit) for more information about the Wiki reorg.

### CODE REVIEW

File: docs/learn/learn-nominator.md [hunk 1]
```diff
@@ -38,171 +38,355 @@ Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more inf
...
-Nominators secure the Relay Chain by selecting good validators and staking DOT.
+If you landed on this page, you decided to understand how you can be a good nominator. Note, this
+page is not for [nomination pool](./learn-nomination-pools.md) members, although pool members might
+gain essential knowledge about how to choose nomination pools.
+
+The information provided on this page is complementary to that on the
+[**Staking Page**](./learn-staking.md) and [**Advanced Staking Page**](./learn-staking-advanced.md).
+Make sure you read those pages as well before nominating.
+
+## Why Nominating?
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
## Why Nominate?
```

File: docs/learn/learn-nominator.md [hunk 2]
```diff
@@ -38,171 +38,355 @@ Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more inf
...
+
+- You become part of the {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} Movement, a
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
- You become part of the {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} movement, a
```

File: docs/learn/learn-nominator.md [hunk 3]
```diff
@@ -38,171 +38,355 @@ Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more inf
...
+  group of professionals and enthusiasts around the world aspiring to build and foster the next-gen
+  Internet, Web 3.0: a decentralized, privacy-focused, and trustless internet.
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
  Internet, Web3: a decentralized, privacy-focused, and trustless internet.
```

File: docs/learn/learn-nominator.md [hunk 4]
```diff
@@ -38,171 +38,355 @@ Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more inf
...
+- You are an essential piece of the puzzle, keeping
+  {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} secure, deciding the future
+  direction of the project by voting in [OpenGov](./learn-opengov.md) and participating in parachain
+  [crowdloans](./learn-crowdloans.md).
```
Comment (profradha) [inline docs/learn/learn-nominator.md]: This statement can be rephrased as it has less to do with nominating DOT/KSM. You can't participate in crowdloans and take part in governance. What you could highlight is here is that "As a nominator securing the network, you can also use the nominated DOT/KSM to participate in Governance"

```suggestion
- You are an essential piece of the puzzle, keeping
  {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} secure, deciding the future
  direction of the project by voting in [OpenGov](./learn-opengov.md) and participating in parachain
  [crowdloans](./learn-crowdloans.md).
```

File: docs/learn/learn-nominator.md [hunk 5]
```diff
@@ -38,171 +38,355 @@ Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more inf
...
+- You start to understand how {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} works
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
- You will start to understand how {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} works
```

File: docs/learn/learn-nominator.md [hunk 6]
```diff
@@ -38,171 +38,355 @@ Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more inf
...
+  at a technical-level. When you feel comfortable with your nomination skills and knowledge, you can
+  open your [nomination pool](./learn-nomination-pools.md), help others secure the network and earn
+  rewards, and build your reputation as a trusted nominator. If you need more than this, the next
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
  rewards, and build your reputation as a trusted nomination pool operator. If you like to be more involved, the next
```

File: docs/learn/learn-nominator.md [hunk 7]
```diff
@@ -38,171 +38,355 @@ Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more inf
...
+  step is to become a [validator](./learn-validator.md).
+- By getting [staking](./learn-staking.md) rewards you fight
+  {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} inflation and (likely) your
+  currency inflation as well.
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
- By getting [staking](./learn-staking.md) rewards you keep up with or (likely) stay ahead of 
  {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} inflation.
```

File: docs/learn/learn-nominator.md [hunk 8]
```diff
@@ -38,171 +38,355 @@ Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more inf
...
+gain essential knowledge about how to choose nomination pools.
+
+The information provided on this page is complementary to that on the
+[**Staking Page**](./learn-staking.md) and [**Advanced Staking Page**](./learn-staking-advanced.md).
+Make sure you read those pages as well before nominating.
+
+## Why Nominating?
+
+- You become part of the {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} Movement, a
+  group of professionals and enthusiasts around the world aspiring to build and foster the next-gen
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
  group of diverse professionals and enthusiasts around the world aspiring to build and foster the next-gen
```

File: docs/learn/learn-nominator.md [hunk 9]
```diff
@@ -38,171 +38,355 @@ Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more inf
...
+staking lock will be visible on the Polkadot-JS UI during the unbonding period, and after it, the
+staking lock can be unlocked, and the bonded funds become free balance you can transfer.
+
+## Setting-up Accounts
+
+### Stash, Controller & Staking Proxy
+
+The first thing you need to do before becoming a nominator is to make sure you have a
+[**stash account**](./learn-staking.md/#stash-and-controller-accounts-for-staking) where you can
+transfer funds you want to use for staking. For these accounts, it is recommended a "cold solution"
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
transfer funds you want to use for staking. For these accounts, it is recommended to use a "cold wallet" solution 
```

File: docs/learn/learn-nominator.md [hunk 10]
```diff
@@ -38,171 +38,355 @@ Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more inf
...
+After setting up the stash account, it is recommended to have a
+[**controller account**](./learn-staking.md/#stash-and-controller-accounts-for-staking) or a
+[**staking proxy**](./learn-staking-advanced.md/#staking-proxies) (or both, controller and staking
+proxy). Although you can be a nominator with just a stash account, having at least a controller or a
+staking proxy is good practice for security reasons.
+
+:::info In the near future, Controller Accounts will be deprecated
+
+The concept of a controller account is very similar to that of a staking proxy: isolate the stash
+account but use its economic power and sign staking-related transactions on behalf of it. This is
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
account but sign staking-related transactions on behalf of it. This is
```

File: docs/learn/learn-nominator.md [hunk 11]
```diff
@@ -38,171 +38,355 @@ Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more inf
...
+:::
+
+## Nominating with the Polkadot-JS UI
+
+### Targets Page
+
+There are many factors to consider when deciding which of your nominations. One helpful tool to
+choose validators is the Staking [Targets](https://polkadot.js.org/apps/#/staking/targets) table in
+the Polkadot-JS UI. This allows sorting validators using various metrics. Below are the relevant
+metrics, followed by a brief description of each.
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
metrics shown as an example, followed by a brief description of each.
```

File: docs/learn/learn-nominator.md [hunk 12]
```diff
@@ -38,171 +38,355 @@ Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more inf
...
+  nominating validators.
+- **total stake**: The total amount of {{ polkadot: DOT :polkadot }}{{ kusama: KSM :Kusama }} tokens
+  staked by nominators and the validator (i.e. own stake, see below).
+- **own stake**: The amount of {{ polkadot: DOT :polkadot }}{{ kusama: KSM :Kusama }} tokens the
+  validator has put up as a stake. A higher own stake can be considered as having more "skin in the
+  game". This can imply increased trustworthiness. However, a validator not having a large amount of
+  "own stake" is not automatically untrustworthy, as the validator could nominate from a different
+  address.
+- **return**: Average annual yield paid out to nominators (i.e. number of rewards divided by the
+  number of bonded tokens). Note that nominating those with a higher yield does not guarantee
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
  number of bonded tokens). Note that nominating those with a higher yield may not guarantee
```

File: docs/learn/learn-nominator.md [hunk 13]
```diff
@@ -38,171 +38,355 @@ Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more inf
...
-  the [bags-list pallet](https://paritytech.github.io/substrate/master/pallet_bags_list/).
-- **active nominator:** a nominator who came out of the NPoS election algorithm backing an active
-  validator. Staking rewards are received by top
-  {{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominatorRewardedPerValidator" defaultValue={512}/> :polkadot }}
-  {{ kusama: <RPC network="kusama" path="consts.staking.maxNominatorRewardedPerValidator" defaultValue={512}/> :kusama }}
-  nominators, but when slashing occurs, all the active nominators backing the validator get slashed.
+:::
 
-![Nominator Election](../assets/staking/nominator-election.png)
+## Good Practices
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
## Good Nominator Practices
```

File: docs/learn/learn-nominator.md [hunk 14]
```diff
@@ -235,199 +419,71 @@ but can be modified via governance. If more than
...
-Finally, if you have a very small amount of DOTs, you may not be able to have your nomination fit
-into the election set. The nominator to validator mapping has to fit in a single block, and if there
-are too many nominators, the lowest-staked nominations will be dropped. This value is obviously
-dynamic and will vary over time. If you review the lowest amount of nominations that are occurring
-on current validators, you can get a good idea of how many DOTs will likely be necessary to have
-your nomination earn you rewards. You can read the blog post
+Finally, if you have a minimal amount of DOTs, you may need more time to fit your nomination into
+the election set. The nominator-to-validator mapping has to fit in a single block, and if there are
+too many nominators, the lowest-staked nominations will be dropped. This value is dynamic and will
+vary over time. You can read the blog post
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
Finally, if you have a minimal amount of DOTs close to the value of `minActiveNomination`, you may need to stake more DOT to get into
the election set. The nominator-to-validator mapping solution needs to be evaluated within a single block duration, and if there are
too many nominators, the lowest-staked nominations will be dropped from even being considered to be part of the electing set. This `minActiveNomination` value is dynamic and will
vary over time. You can read the blog post
```

**Review by DrW3RK — CHANGES_REQUESTED** [review]

**Comment by filippoweb3** [comment]
hey @DrW3RK let me know if you have any additional feedback about this PR ;)

**Review by DrW3RK — APPROVED** [review]

**Comment by DrW3RK** [comment]
Good to go!

**Pull request merged**